netket/models/rbm.py

```diff
@@ -127,3 +127,72 @@ def __call__(self, x):
         im = jnp.sum(im, axis=-1)
 
         return re + 1j * im
+
+
+class RBMSymm(nn.Module):
+    """A restricted boltzman Machine, equivalent to a 2-layer FFNN with a
+    nonlinear activation function in between
+
+    Attributes:
+        dtype: dtype of the weights.
+        activation: The nonlinear activation function
+        alpha: feature density. Number of features equal to alpha * input.shape[-1]
+        use_bias: if True uses a bias in the dense layer (hidden layer bias)
+        use_visible_bias: if True adds a bias to the input
+        kernel_init: initializer function for the weight matrix.
+        bias_init: initializer function for the bias.
+        visible_bias_init: initializer function for the visible_bias.
+    """
+
+    permutations: Callable
+    dtype: Any = np.float64
+    activation: Any = nknn.logcosh
+    alpha: Union[float, int] = 1
+    use_bias: bool = True
+    use_visible_bias: bool = True
+
+    kernel_init: Callable[[PRNGKey, Shape, Dtype], Array] = normal(stddev=0.1)
+    bias_init: Callable[[PRNGKey, Shape, Dtype], Array] = normal(stddev=0.1)
+    visible_bias_init: Callable[[PRNGKey, Shape, Dtype], Array] = normal(stddev=0.1)
+
+    @nn.compact
+    def __call__(self, x_in):
+        x = nknn.DenseSymm(
```

Comment:
is this calling the class or the class constructor?

Reply:
The constructor.